ui/job-view/details/DetailsPanel.jsx

```diff
@@ -173,6 +180,7 @@ class DetailsPanel extends React.Component {
           phSeriesPromise,
           jobArtifactsPromise,
           builtFromArtifactPromise,
+          taskDefinitionPromise,
```

Comment:
Hi @camd 
Adding the extra promise in here makes one of the Selenium tests to time out.
Before this test failed I noticed on Friday this pattern.

I believe a better pattern is to mark a tab as disabled and then let the component within that tab notify the parent TabsPanel.

In this PR, I changed the model and it fixes the test:
https://github.com/mozilla/treeherder/pull/6645